templates/security/cves/about.html

```diff
@@ -0,0 +1,438 @@
+            <a href="https://help.ubuntu.com/community/AutomaticSecurityUpdates">enable unattended upgrades.</a>
+          </p>
+
+          <h3>Update manually</h3>
+          <p>
+            You can also get patches as soon as they become available by upgrading all your installed packages to the latest version. You can do this by running
+            the following command in your terminal:
+          </p>
+          <p>
+            <pre><code>sudo apt update && sudo apt upgrade</code></pre>
+          </p>
+          <p>
+            We recommend not to cherry-pick updates from individual packages. If no fix is available yet for a specific CVE, you can check if there is any mitigation
+            or further information in the notes of the CVE page.
+          </p>
+
+          <h3>Get advanced security coverage with Ubuntu Pro</h3>
+          <p>
+            On top of the five years of free standard support for LTS releases, you can get <a href="/security/esm">Expanded Security Maintenance (ESM)</a> with an
+            <a href="/pro">Ubuntu Pro</a> subscription. ESM provides 10 years of security updates for Ubuntu Main packages and 23,000+ Ubuntu Universe packages,
+            including additional security updates from  Canonical for critical and high priority CVEs. Ubuntu Pro is free for personal use on up to 5 machines, or 50 machines for active Ubuntu community members.
+          </p>
+          <p>
+            Ubuntu Pro also includes <a href="/security/livepatch">live kernel patching with Livepatch</a>, <a href="/security/fips">FIPS compliant security updates</a>,
+            <a href="/robotics/ros-esm">extended support for ROS</a>, and more.
+          </p>
+
+          <h3>Fix a specific CVE with the Ubuntu Pro Client</h3>
+          <p>
+            If you want to get the fixes for a specific CVE, make sure you have the
+            <a href="https://canonical-ubuntu-pro-client.readthedocs-hosted.com/en/latest/tutorials/fix_intro/">Ubuntu Pro Client installed, updated and set up</a>.
+            Then run this command on your terminal, replacing CVE-YYYY-XXXX with a valid CVE ID:
+          </p>
+          <p>
+            <pre><code>sudo pro fix CVE-YYYY-XXXX</code></pre>
+          </p>
+          <p>
+            We recommend not to cherry-pick updates from individual packages. If no fix is available yet for a specific CVE, you can check if there is any mitigation
+            or further information in the notes of the CVE page.
+          </p>
+        </div>
+
+        <h2 class="question-heading" id="repos">Repositories</h2>
+        <div class="p-section">
+          <p>
+            Security updates get distributed through several repositories. Some of them are
+            <a href="https://help.ubuntu.com/community/Repositories/Ubuntu">public and freely available</a>, while some others require an
+            <a href="/pro">Ubuntu Pro subscription</a>.
+          </p>
+          <table>
```

Comment:
These tables are split up a bit differently on the doc 
![image](https://github.com/canonical/ubuntu.com/assets/17607612/bc6f223c-e314-4a08-aaf9-29d4db7f8c0d)